astroquery/astrometry_net/core.py

```diff
@@ -33,6 +33,24 @@
 __all__ = ['AstrometryNet', 'AstrometryNetClass']
 
 
+MISSING_API_KEY = """
+Astrometry.net API key not set. You should either set this in the astroquery configuration file using:
+
+    [astrometry_net]
+    api_key = qwdqwjnoi12ioj
```

Comment:
Would it be better to use something that is obviously not a key here? I assume the value here isn't a valid key
but something like `XXX` or `FILL_IN_YOUR_KEY` might be better.

Reply:
Ok sounds good, I can do that